cluster-api/src/test/java/io/aiven/klaw/clusterapi/ClusterApiControllerIT.java

```diff
@@ -404,6 +411,52 @@ public void createTopicsWhenTopicAlreadyExists() throws Exception {
         });
   }
 
```

Comment:
What do we expect to happen if the same schema is assigned to two different topics?
Should that be deleted?

Reply:
With Subject naming strategy, it's always one schema with multiple versions for a topic. And on deletion, all the associated versions of the topic would be deleted.